mesa/space.py

```diff
@@ -989,22 +986,20 @@ def is_cell_empty(self, node_id: int) -> bool:
         return not self.G.nodes[node_id]["agent"]
 
     def get_cell_list_contents(self, cell_list: list[int]) -> list[GridContent]:
-        """Returns the contents of a list of cells ((x,y) tuples)
-        Note: this method returns a list of `Agent`'s; `None` contents are excluded.
+        """Returns the agents in a list of nodes ((x,y) tuples)
+        Note: this method returns a list of `Agent`.
```

Comment:
I haven't checked, but isn't the old version more correct? IIRC excluding `None`'s is a general behavior of `get_cell_list_contents` in various grid implementations.

Reply:
the problem is that from what I understand it checks it checks if the list is empty, there is no None default value in NetworkGrid, this would be confusing, and it is clear I think that if a list is empty it doesn't have any agent so no need to specify that

Reply:
This docstring seems to be copy-pasted from `Grid.get_cell_list_contents`. It shouldn't have an `(x, y)` coordinate either, but a node ID instead. But at least, it should say nodes with empty content are excluded -- if this is excluded, it would be confusing.

Reply:
I meant, if the statement that "nodes with empty content are excluded" is not said in the docstring, it would be confusing.

Reply:
good catch! done